config/routes.rb

```diff
@@ -1,11 +1,13 @@
 Rails.application.routes.draw do
 
+
   devise_for :users
   resources :users, only: [:update]
    
-   resources :topics do
-     resources :posts, except: [:index]
-   end
+   resources :topics, :posts, :comments do
+   resources :posts, except: [:index]
+   resources :comments, only: [:create]
```

Comment:
I tried to do resources :posts, :comments except: [:index]
But then I run into an error when I run rails s

/Users/Dru/.rvm/gems/ruby-2.1.5/gems/activesupport-4.1.6/lib/active_support/dependencies.rb:241:in `load': /Users/Dru/code/Bloccit/config/routes.rb:8: syntax error, unexpected tIDENTIFIER, expecting keyword_end (SyntaxError)
   resources :posts, :comments except: [:index]